hedera-node/test-clients/src/main/resource/contract/contracts/MintContract/MintContract.bin

```diff
@@ -1 +1 @@
60039290920b9a90995090975095505050505050565b604080516001600160a01b038681166024830152858116604483015284166064820152600783900b6084808301919091528251808303909101815260a490910182526020810180516001600160e01b03167feca36917000000000000000000000000000000000000000000000000000000001790529051600091829182916101679161065791906109a1565b6000604051808303816000865af19150503d8060008114610694576040519150601f19603f3d011682016040523d82523d6000602084013e610699565b606091505b5091509150816106aa5760156106be565b808060200190518101906106be91906109cf565b60030b979650505050505050565b634e487b7160e01b600052604160045260246000fd5b604051601f8201601f1916810167ffffffffffffffff8111828210171561070b5761070b6106cc565b604052919050565b600067ffffffffffffffff82111561072d5761072d6106cc565b5060051b60200190565b6000601f838184011261074957600080fd5b8235602061075e61075983610713565b6106e2565b82815260059290921b8501810191818101908784111561077d57600080fd5b8287015b8481101561081457803567ffffffffffffffff808211156107a25760008081fd5b818a0191508a603f8301126107b75760008081fd5b858201356040828211156107cd576107cd6106cc565b6107de828b01601f191689016106e2565b92508183528c818386010111156107f55760008081fd5b8181850189850137506000908201870152845250918301918301610781565b50979650505050505050565b60006020828403121561083257600080fd5b813567ffffffffffffffff81111561084957600080fd5b61085584828501610737565b949350505050565b80356001600160a01b038116811461087457600080fd5b919050565b8060070b811461088857600080fd5b50565b6000806000606084860312156108a057600080fd5b6108a98461085d565b92506108b76020850161085d565b915060408401356108c781610879565b809150509250925092565b6000602082840312156108e457600080fd5b6108ed8261085d565b9392505050565b6000806040838503121561090757600080fd5b6109108361085d565b9150602083013567ffffffffffffffff81111561092c57600080fd5b61093885828601610737565b9150509250929050565b67ffffffffffffffff8116811461088857600080fd5b60006020828403121561096a57600080fd5b81356108ed81610942565b60005b83811015610990578181015183820152602001610978565b838111156101355750506000910152565b600082516109b3818460208701610975565b9190910192915050565b8051600381900b811461087457600080fd5b6000602082840312156109e157600080fd5b6108ed826109bd565b634e487b7160e01b600052603260045260246000fd5b6000606082016001600160a01b0386168352602067ffffffffffffffff8616818501526060604085015281855180845260808601915060808160051b870101935082870160005b82811015610a8d57878603607f1901845281518051808852610a6e81888a01898501610975565b601f01601f191696909601850195509284019290840190600101610a47565b50939998505050505050505050565b600080600060608486031215610ab157600080fd5b610aba846109bd565b9250602080850151610acb81610942565b604086015190935067ffffffffffffffff811115610ae857600080fd5b8501601f81018713610af957600080fd5b8051610b0761075982610713565b81815260059190911b82018301908381019089831115610b2657600080fd5b928401925b82841015610b4d578351610b3e81610879565b82529284019290840190610b2b565b8095505050505050925092509256fea2646970667358221220539021dabefb3ad6b4b014421b748d2f5430fab6b0b0f31c72a38f86f0eabb5d64736f6c63430008090033
```

Comment:
The change here is that I've removed **0x** at the beginning of the file. The reason for that is when we call `EthereumCallDataHydration.tryToHydrate` we try to decode the hex `Hex.decode(callDataFile.contents().toByteArray())` but the **0x** part is not hex. I saw that in most of our **.bin** files we don't have that so I think it should be removed.

I have two questions:
1. Should I remove the 0x prefix for all *.bin files we have that(`AssociateDissociate.bin, Associator.bin, MixedMintToken.bin, and NastedAssociateDissociate.bin`)?
2. In `EthereumCallDataHydration.tryToHydrate` when we try to decode the hex `Hex.decode(callDataFile.contents().toByteArray())` if there is an exception we throw `INVALID_FILE_ID`. Do you guys think there is a more suitable error we can throw there? It's a bit misleading, at least in my case.